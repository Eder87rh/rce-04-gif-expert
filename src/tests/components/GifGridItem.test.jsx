import { shallow } from "enzyme";
import React from "react";
import GifGridItem from "../../components/GifGridItem";
import toJson from "enzyme-to-json";

describe("GidGridItem.jsx", () => {
  it("Component should display correctly", () => {
    const wrapper = shallow(<GifGridItem id="1" title="title" url="url" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
