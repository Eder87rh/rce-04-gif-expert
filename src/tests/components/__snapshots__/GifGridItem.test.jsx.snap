// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GidGridItem.jsx Component should display correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="title"
    src="url"
  />
  <p>
    title
  </p>
</div>
`;
